src/server/store/ReplyStore.ts

```diff
@@ -30,25 +29,29 @@ export class ReplyStore {
     return store
   }
 
-  async add (reply: APActivity): Promise<void> {
+  async add (reply: AnyAPObject): Promise<void> {
     if (reply.id === undefined) {
       throw new Error('Reply ID is missing.')
     }
-    const store = this.forPost(reply.id)
-    await store.add(reply)
-  }
 
-  async remove (url: string): Promise<void> {
-    const store = this.forPost(url)
-    await store.remove(url)
-  }
+    // Check if the reply has 'object' property and is not an array.
+    if (!('object' in reply) || Array.isArray(reply.object) || typeof reply.object !== 'object') {
```

Comment:
This is no longer necessary since the `reply` is already the `activity.object`

Reply:
Hmm, after adjusting the code to treat the reply itself (and not reply.object) as the main object of interest, and checking directly for the 'inReplyTo' property within the reply, our tests fail with the error indicating that the "reply does not contain an inReplyTo property."

Reply:
In the replystore tests We should be adding the reply.object to the store instead of the entire activity for the reply.

Could you also an add a test to the APSystem tests which ingests a new Create activity with a reply? This file can focus on storing the replies themselves and the APSystem test can tst that we're handling the activity correctly befoe storing

Reply:
Done

Reply:
Apologies. Strangely, on my machine, linter didn't show any errors in the console after running `npm run lint`. I will fix this ASAP, including both lint errors and apsystem test changes.